contents/customers/ycombinator.md

```diff
@@ -0,0 +1,48 @@
+
+"Many people find Y Combinator to be the the most productive period of their lives, which is definitely what we found”, said PostHog CEO James Hawkins in [a blogpost he wrote about his Y Combinator experience](/blog/moving-to-sf). Now, PostHog is used by Y Combinator to improve many of the products we benefitted from! 
+
+“We use PostHog for our [Startup Schoo](https://www.startupschool.org/)l, for our public [YC Startup Library](https://www.ycombinator.com/library), and for a product called [Co-Founder Matching](https://www.ycombinator.com/cofounder-matching),” said Cat Li, Product and Engineering Lead for Y Combinator Startup School. “PostHog is especially useful for Co-Founder Matching, which is almost like a dating app for co-founders.”
+
+### How Y Combinator gathers 30% more data with PostHog than Google Analytics
+
+PostHog isn’t the only tool in Y Combinator’s stack, of course. Teams across the organization can each have their own processes, tools and OKRs to guide their progress. Some teams at Y Combinator use Segment or Amplitude — but for the flagship Startup School project, PostHog was a clear choice. 
+
+“We were very thoughtful about what we wanted to use with Startup School and Co-founder Matching,” said Cat. “We chose PostHog so we wouldn’t lose a bunch of the traffic and data. We found that with Google Analytics and other platforms we would lose up to 30% of our data due to adblockers and users who would not allow third-party tracking. Our users are very tech-savvy... very likely to block third-party cookies.”
+
+“Because Startup School is very wide reaching, it really matters to us that we have an accurate understanding of what's happening. That meant PostHog was the obvious choice.”
+
+In addition to capturing more data, PostHog also provided a wider array of tools than other platforms, with multiple insight types, correlation analysis, and session recordings enabling Y Combinator to explore data from multiple approaches. 
+
+“Other tools, like Google Analytics, just weren’t robust enough,” said Cat. “As an engineer, I didn’t feel they offered enough detail or control. One thing we really liked about PostHog was that we had control over everything. We could autocapture a lot of events just by using the snippet, but we could also configure custom events too which was great.”
+
+### How Y Combinator runs experiments to build better products
+
+The level of control PostHog offers extends beyond just instrumentation, however. Cat’s team is also able to use the entire suite of insight tools to choose the appropriate key metric for each product. For Y Combinator Startup School, for example, the team uses [trends insights](/manual/trends) to monitor weekly users and engagement — while [funnels](/manual/funnels) are more useful for Co-Founder Matching because the product involves many discrete steps. 
```

Comment:
```suggestion
Because Y Combinator has deployed PostHog across multiple products, Cat’s team uses the entire suite of insight tools to monitor the appropriate key metric for each product. For Y Combinator Startup School, for example, the team uses [trends insights](/manual/trends) to monitor weekly users and engagement — while [funnels](/manual/funnels) are more useful for Co-Founder Matching because the product involves many discrete steps. Insights are collected into dashboards for ease, along with additional dashboards which report on running experiments. 
```